Packs/FeedCofense/Integrations/FeedCofense/FeedCofense.py

```diff
@@ -0,0 +1,311 @@
+            list -- List of dicts representing indicators.
+
+        Examples:
+            >>> Client.process_item({"id": 123, "blockSet": [{"data_1": "ip", "blockType": "IPv4 Address"}]})
+            [{'value': 'ip', 'type': 'IP', 'rawJSON': \
+{'data_1': 'ip', 'blockType': 'IPv4 Address', 'value': 'ip', 'type': 'IP', 'threat_id': 123}}]
+        """
+        results = list()
+        block_set: List[dict] = threat.get("blockSet", [])
+        thread_id = threat.get("id")
+        for block in block_set:
+            indicator, value = cls._convert_block(block)
+            block["value"] = value
+            block["type"] = indicator
+            block["threat_id"] = thread_id
+            if indicator:
+                results.append({"value": value, "type": indicator, "rawJSON": block})
+
+        return results
+
+
+def test_module(client: Client) -> str:
+    """A simple test module
+
+    Arguments:
+        client {Client} -- Client derives from BaseClient
+
+    Returns:
+        str -- "ok" if succeeded, else raises a error.
+    """
+    client.build_iterator()
+    return "ok"
+
+
+def fetch_indicators_command(
+        client: Client,
+        begin_time: Optional[int] = None,
+        end_time: Optional[int] = None,
+        limit: Optional[int] = None,
+) -> Union[Dict, List[Dict]]:
+    """Fetches the indicators from client.
+
+    Arguments:
+        client {Client} -- Client derives from BaseClient
+        indicator_type {str} -- [description]
+
+    Keyword Arguments:
+        update_context {bool} -- Should update context (default: {False})
+        limit {int} -- Maximum indicators to fetch (when update_context is True)
+        start_time {Optional[Union[str, int, dict]]} -- Last run object from Demisto or timestamp. (default: {None})
+
+    Returns:
+        Union[Dict, List[Dict]] -- [description]
+    """
+    indicators = list()
+    for threat in client.build_iterator(begin_time=begin_time, end_time=end_time):
+        # get maximum of limit
+        new_indicators = client.process_item(threat)
+        indicators.extend(new_indicators)
+        if limit and limit < len(indicators):
+            indicators = indicators[:limit]
+            break
+    return indicators
+
+
+def fetch_indicators_builder(client: Client, fetch_time: str) -> None:
+    """Build the fetch_indicators and saves timestamp to lastRun
+
+    Args:
+        client: Client derives from BaseClient
+        fetch_time: fetch time (for example: "3 days")
+    """
+    last_fetch = demisto.getLastRun()
+    if isinstance(last_fetch, dict) and "timestamp" in last_fetch:
+        begin_time = last_fetch.get("timestamp")
+        end_time = get_now()
+    else:  # First fetch
+        begin_time, end_time = parse_date_range_no_milliseconds(fetch_time)
+    indicators = fetch_indicators_command(client, begin_time, end_time)
+    # Send indicators to demisto
+    for b in batch(indicators, batch_size=2000):
+        demisto.createIndicators(b)
```

Comment:
Please remove and use the one you added to `CommonServerPython`.

Reply:
this is actually using the batch in commonsp